### PR TITLE
[DOC] Add `fMRIPrep` to glossary

### PR DESCRIPTION
**Pull request by NicolasGensollen**
Small PR to add `fMRIPrep` to the glossary and try triggering the auto-comment action (See #2982).
Ping @thomasbazeille

### CODE REVIEW

File: doc/glossary.rst [hunk 1]
```diff
@@ -71,6 +71,16 @@ If you wish to add a missing term, please `create a new issue`_ or
         The signal picked up by the MRI scanner is sensitive to these
         modifications of the local magnetic field.
 
+    fMRIPrep
+        `fMRIPrep`_ is a :term:`fMRI` data preprocessing pipeline designed
+        to provide an interface robust to variations in scan acquisition
+        protocols with minimal user input. It performs basic processing
+        steps (coregistration, normalization, unwarping, noise component
+        extraction, segmentation, skullstripping etc.) providing outputs
+        that can be easily submitted to a variety of group level analyses,
```
Comment (bthirion) [inline doc/glossary.rst:80]: these outputs are often called confounds or nuisance parameters.

**Review by bthirion — COMMENTED** [review]
LGTM otherwise.

File: doc/glossary.rst (hunk 1 above)
Comment (NicolasGensollen) [reply doc/glossary.rst]: Thanks!

**Review by bthirion — APPROVED** [review]
Thx.

**Comment by NicolasGensollen** [comment]
Thanks, merging...

**Pull request merged**